src/js/gi/reducers/profile.js

```diff
@@ -1,12 +1,23 @@
 import { FETCH_PROFILE_STARTED, FETCH_PROFILE_FAILED, FETCH_PROFILE_SUCCEEDED } from '../actions';
-import { setPageTitle } from '../actions';
 
 const INITIAL_STATE = {
   attributes: {},
   version: {},
   inProgress: false,
 };
 
+function normalizedAttributes(attributes) {
+  const name = attributes.name ? attributes.name.toUpperCase() : attributes.name;
+  const city = attributes.city ? attributes.city.toUpperCase() : attributes.city;
+  const state = attributes.state ? attributes.state.toUpperCase() : attributes.state;
```

Comment:
For these, you could just do something like:

`const name = attributes.name && attributes.name.toUpperCase()`.